lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -92,34 +94,192 @@ Function CheckModified-File($module, $url, $dest, $headers, $credentials, $timeo
     }
 
     if ($webRequest -is [System.Net.FtpWebRequest]) {
-        $webRequest.Method = [System.Net.WebRequestMethods+Ftp]::GetDateTimestamp
+        $webRequest.Method = [System.Net.WebRequestMethods+Ftp]::DownloadFile
     } else {
-        $webRequest.Method = [System.Net.WebRequestMethods+Http]::Head
+        $webRequest.Method = [System.Net.WebRequestMethods+Http]::Get
     }
 
     # FIXME: Split both try-statements and single-out catched exceptions with more specific error messages
     Try {
-        $webResponse = $webRequest.GetResponse()
-        $webLastMod = $webResponse.LastModified
+         # TBD implement case for FTP
+         $webResponse = $webRequest.GetResponse()      
+         $requestStream = $webResponse.GetResponseStream()
```

Comment:
perhaps '$responseStream' would be a better variable name here instead of '$requestStream' ?

Reply:
Fixed